### ISSUE TITLE
Add invert mouse option

### ISSUE THREAD

**Issue by Flamefire** [Return-To-The-Roots/s25client#951]
Requested here: https://www.siedler25.org/index.php?com=forum&mod=forum&action=thread&id=1572&start=-1#last

There should be an option to invert the direction for map scrolling.

@Spikeone Which setting is "inverted"? Mouse moves right -> Map moves left (you see more on right) as it is currently and in S2 or the other way round?

Note: This function already exists, it is just missing an option entry

**Comment by Spikeone** [comment]
Well - inverted is the opposite of the current direction? Normal is: Move mouse right, scroll right, so inverted is move mouse right, scroll left

**Comment by Flamefire** [comment]
This doesn't quite answer it.

Currently you basically move the camera/window over the map. Or you move the map inversed to the mouse (which is the same). You could also expect to move the map with the mouse and not the camera. Do you know of any other RTS games with "inverted" mouse setting? What is the "normal" there?

My point: "Inverted" could be the default and you can disable it. Or the other way round.

**Comment by Spikeone** [comment]
C&C Zero Hour - like RttR
Civ5 - inverted to RttR
OttD - like RttR

Any other strategy game I've got installed doesn't use scrolling by holding right or leftclick.

I don't think one can say that any modern game does use an option that is inverted to RttR. Although I have to agree that a lot of people use their smart phones quite a lot - swipe left to scroll right it is as you kinda anchor one point and shift it away so you see more the opposite direction.

I'd still leave it Settlers 2 default but at the option. Maybe we should add a startup window (like for debug data) which is shown on first start introducing some of our new features? Or at least hinting where you could find them.

**Comment by Flamefire** [comment]
> the default shouldn't be changed

This was never up for discussion. We can call the current handling "inverted" and make it the default.

Unless there is someone against this I'd call the current handling "Normal" and the other "Inverted" as it seems to be impossible to get a standard definition what "Normal" is. So I'll define "Normal" as "as in S2 original"

**Comment by Spikeone** [comment]
@Flamefire 

> My point: "Inverted" could be the default and you can disable it. Or the other way round.

Sorry, got that wrong then. I actually think that in newer games (if they support click scrolling) Settlers 2 seems inverted - but then again whats a new game and when do they become old. If we'd like to stay save just call it "original" and "inverted" that way everybody should understand, that inverted means inverted to the original. It's like in some games that include airplanes you can fly - sometimes you have to invert controls, sometimes you don't so there is no real normal, just as you said.

**Comment by Hezkore** [comment]
This was never implemented, right?
I'd really like to "invert" the camera scrolling, so it feels like I'm dragging the camera around.
I'd also like an option for the camera scroll speed, and that the speed scales with the current zoom level.

**Comment by theKalash** [comment]
I just got back into Settlers II the other day, bought the History Edition but was quickly disapointed when I found out they still haven't fixed ships. 

After trying RTTR the first thing I noticed what that scrolling was inverted (and too fast), so I guess +1 for this enhancement. 
I'm not sure which direction the original S2 used, but compared to the HE (and most of my other RTS expirences), the RTTR scrolling is "inverted" for me.
